Sources/Testing/Support/Versions.swift

```diff
@@ -141,30 +141,25 @@ let swiftStandardLibraryVersion: String = {
 // MARK: - sysctlbyname() Wrapper
 
 #if !SWT_NO_SYSCTL && SWT_TARGET_OS_APPLE
-/// Get a value by calling `sysctlbyname()`.
+/// Get a string value by calling `sysctlbyname()`.
 ///
 /// - Parameters:
 ///   - name: The name of the value to get, such as `"kern.osversion"`.
 ///
-/// - Returns: An array containing the bytes of the requested value, or `nil`
-///   if the value could not be read.
-///
-/// This function does not convert the result to a `String` since not all
-/// `sysctl()` values are strings. To convert the result to a string, use
-/// `String(validatingUTF8:)`.
+/// - Returns: A string containing the requested value interpreted as a UTF-8
+///   string, or `nil` if the value could not be read or could not be
+///   interpreted as UTF-8.
 ///
 /// This function is not part of the public interface of the testing library.
-func sysctlbyname(_ name: String) -> [CChar]? {
+func sysctlbyname(_ name: String, as _: String.Type) -> String? {
```

Comment:
Worth making private?
```suggestion
private func _sysctlbyname(_ name: String, as _: String.Type) -> String? {
```